travis/upload_build_results_to_gcs.py

```diff
@@ -125,22 +132,48 @@ def _DecryptGCPServiceFileTo(service_file_path: str):
             e.__class__.__name__, redacted_message))
 
 
-def _UploadBuildResults(gcs_bucket: storage.Bucket, gcs_build_results_dir: str):
-  """Uploads all build results to Google Cloud Storage."""
-  logging.info("Will upload build results to gs://%s/%s.",
-               os.environ[_GCS_BUCKET], gcs_build_results_dir)
-
-  for build_result in os.listdir(flags.FLAGS.build_results_dir):
-    path = os.path.join(flags.FLAGS.build_results_dir, build_result)
+def _UploadDirectory(local_dir:str, gcs_bucket: storage.Bucket, gcs_dir: str):
```

Comment:
Nit: missing space after `local_dir:`.

Reply:
Thanks for pointing this out! I added it.